tests/release_tests/test_launch/launch-config.yaml

```diff
@@ -0,0 +1,13 @@
+base_url: https://api.wandb.ai
+max_jobs: -1
```

Comment:
could having uncapped # of jobs overwhelm the resource? I know for sweeps this can easily happen, especially on small compute instances. Capping at 8 might be better as a default.